sentinelsat/sentinel.py

```diff
@@ -1097,6 +1097,42 @@ def check_bounds(geometry):
     wkt = re.sub(r'(?<!\d) ', '', wkt)
     return wkt
 
+def placename_to_wkt(placename):
+    """Geocodes the placename to rectangular bounding extents using Nominatim API and 
+       returns the corresponding 'ENVELOPE' form Well-Known-Text.
+   
+    Parameters
+    ----------
+    placename : str
+        the query to geocode
+    
+    Returns
+    -------
+    ENVELOPE coordinates 
+        wkt string in form 'ENVELOPE(minX, maxX, maxY, minY)'
+    """
+
+    rqst = requests.post('https://nominatim.openstreetmap.org/search', \
+                         params = { 'q':placename,'format':'geojson'})
+    #Check that the response from Openstreetmapserver has status code 2xx 
+    #and that the response is valid JSON.
+    _check_scihub_response(rqst)
+    jsonlist = rqst.json()
+    try:
+        #Get the First result's bounding box.
+        bbox = jsonlist['features'][0]['bbox']
+    except IndexError:
+        #bbox is empty
+        raise Exception("No match found for entered Placename query!")
+    except:       
```

Comment:
Avoid overly broad exception catching. Re-raising with a generic error message does not add anything useful.

Reply:
I cannot think of any other scenario where any other exception can result since most of them are handled by _check_scihub_response(). Do you want to check for valid bbox coordinates too? I require help on this one.